### DISCUSSION TITLE
Rethink more-info dialog

### DISCUSSION THREAD

**Discussion by BoneheadFraggle** [home-assistant/frontend#7515] [Other feature requests]
Background:
I want to think of Home Assistant as a something that is user friendly, easy to consume and makes everyday things easier, better, more informative. From that perspective, I always prefer "less is more" when it comes to displaying information and details. Keep all data, but keep the relevant data in the user's focus.

As an admin on the other hand, I also want it to be reasonably easy to maintain and administer.
But being an admin, I can live with a lot less polished design and "more clicks" to achieve the goal. 

So, primarily, I want it to be nice and fast to use for users, guests, family.
And even I see myself as a family member most of the time, and an admin only when needed.
I even use a standard user account for my daily usage, just to verify all functionality - and the admin user for configuring and beta testing.

With that in mind, I'd like to suggest a few changes in the more-info with the goal to gain a cleaner and more consistent experience:

- Only show basic, but relevant information on the first tab. 

- Move all "extra-info/admin/superuser/I need that attribute" data to the second tab.

- Maybe even consider a third tab for attributes

- The first tab should always be visible. 

- The "extra-info/admin/superuser/I need that attribute"-tab(s) should only show if you're an admin

What is relevant data then? I think a temperature graph is relevant. Both family and guests like to see how cold it was outside during the night. Attributes however, are not relevant (ever?), the "extra-info/admin/superuser/I need that attribute"-tab.

- Logbook is nice to have in the more-info, but for me it is "over-information" and Logbook is something that's can invade privacy and should therefore be moved to second tab. 

- Today the Logbook sometimes shows up on the first (only) tab, and sometimes on the second. 
_A few examples of domains that have Logbook on the first (only) tab: binary_sensor, counter, input_boolean, input_slider, input_text, alarm, switch.., while these show the Logbook in the second tab: media_player, light (even if it is a light switch), script, weather.._
We should make it consistent by always having the Logbook on the second tab, the "extra-info/admin/superuser/I need that attribute"-tab.

Another thing that should move to the second tab is actionable buttons in the more-info card. For example the ```counter:```, where (increment, decrement, reset). I am sure there are others.
Motive: It's fine for a guest to see how many cycles our washing-machine has, but I don't want him to change or even reset it, on purpose or not. That should be an admin thing to do.

- Naming. Today we have Details and History. Since I suggest we use the History tab for other data then history, a name change might be in order. 
A few alternatives:
1. Details > Info and History > Details
2. Details > Status and History > Details
3. Details > Status and History > More info

An example using Details > Status and History > Details:

Light entity today:
![image](https://user-images.githubusercontent.com/2287810/97488938-35322b80-195f-11eb-9ca3-717a0ffbd262.png)

Light entity suggestion mock up (Status):
![image](https://user-images.githubusercontent.com/2287810/97488984-42e7b100-195f-11eb-9969-9792491051b7.png)

Light entity suggestion mock up (Details):
![image](https://user-images.githubusercontent.com/2287810/97489008-4aa75580-195f-11eb-98ce-2cf127ee109a.png)

Feel free to pitch in. What do you think?

**Comment by iantrich** [comment]
I agree with most presented here, but I think your counter use case is a little too specific. You may absolutely want a user to be able to interact with a counter.

**Reply by BoneheadFraggle** [reply]
Thinking about it: I agree.
Better to do template sensor if you just want to show the value without option to change it.
